tests/mappings/mapping_q_inverse_quadratic_approximation.output

```diff
@@ -0,0 +1,355 @@
+
+DEAL::Testing 2D with point -0.1498310826 0.7854420410
+DEAL::Testing on cell 0_1:0 with center 0.8227241336 0.4750000000
+DEAL::Exact inverse:                   1.679734433 2.008122269
+DEAL::Affine approximation:            1.294871897 6.963559088
+DEAL::Inverse quadratic approximation: 1.644896745 2.003800318
```

Comment:
That's a pretty good approximation!

Reply:
A quarter of a annulus is also something that is approximated badly with an affine approximation whereas a quadratic function captures almost all of it.